app/services/daily_digest_scheduler_service.rb

```diff
@@ -0,0 +1,47 @@
+class DailyDigestSchedulerService
+  attr_reader :digest_range_hour
+  attr_accessor :starts_at, :ends_at
+
+  LOCK_NAME = "daily_digest_scheduler".freeze
+
+  def self.call(*args)
+    new.call(*args)
+  end
+
+  def initialize
+    @digest_range_hour = ENV["DIGEST_RANGE_HOUR"] || 8
```

Comment:
I think you'll have to do `ENV.fetch("DIGEST_RANGE_HOUR", "8").to_i` to make sure it's an integer.

Reply:
It gets stringificatorised in the `configured_(starts|ends)_at` methods anyway so it probably doesn't matter much. I've updated it in 46a59e7 as having a predictable type is a noble cause 😄 I've also fixed it so as it's easier on @kevindew's eyes 🤣